### ISSUE TITLE
Input validators empty array will return the form v-model false

### ISSUE THREAD

**Issue by rainrain001** [antoniandre/wave-ui#46]
I tried to create a shared input component using `<w-input>`  with a props validators. If the prop validators from `<w-input>` received empty array `[]` the form v-model will return false. just like this code below: **Note: just imagine `<w-input>` is my shared input component.

```
<template>
  <w-form v-model="isFormValid" @submit="submit">
    <w-input
      v-model="contact.firstname"
      type="text"
      label="First name"
      required
      :validators="[validators.required]"
    />
    <w-input
      v-model="contact.lastname"
      type="text"
      label="Last name"
      :validators="[]"
    />
    <w-button type="submit" class="mr1">
      Submit
    </w-button>
  </w-form>
</template>

<script lang="ts">
import { defineComponent, watchEffect, ref } from 'vue'

const TestPage = defineComponent({
  name: 'TestPage',
  setup () {
    const isFormValid = ref(null)
    const contact = ref({
      firstname: '',
      lastname: ''
    })
    const validators = {
      required: value => !!value || 'This field is required',
    }
    function submit () {
      console.log('Submit', isFormValid.value)
    }
    watchEffect(() => {
      console.log(isFormValid.value)
    })
    return { isFormValid, contact, validators, submit }
  }
})
export default TestPage
</script>
```

**Comment by antoniandre** [comment]
Thanks for your feedback @rainrain001! 
I'll have a look, I agree an empty array of validators should not invalidate the field nor the form (if I understand well).

**Comment by antoniandre** [comment]
Hi @rainrain001, this is now fixed in version 1.25.2.
Thanks for reporting! 👌 
Please reopen if not solved.
